test/e2e/cypress/integration/clusters_overview.js

```diff
@@ -44,5 +46,193 @@ context('Clusters Overview', () => {
         cy.get(`a:contains(${clusterID})`).should('be.visible');
       });
     });
+
+    describe('Health status for each cluster is correct', () => {
+      before(() => {
+        cy.selectChecks(
+          clusterIdByName(healthyClusterScenario.clusterName),
+          healthyClusterScenario.checks
+        );
+        cy.requestMockRunnerResult(healthyClusterScenario.result);
+        cy.requestChecksExecution(
+          clusterIdByName(healthyClusterScenario.clusterName)
+        );
+
+        cy.selectChecks(
+          clusterIdByName(unhealthyClusterScenario.clusterName),
+          healthyClusterScenario.checks
+        );
+        cy.requestMockRunnerResult(unhealthyClusterScenario.result);
+        cy.requestChecksExecution(
+          clusterIdByName(unhealthyClusterScenario.clusterName)
+        );
+      });
+
+      after(() => {
+        cy.selectChecks(
+          clusterIdByName(healthyClusterScenario.clusterName),
+          []
+        );
+
+        cy.selectChecks(
+          clusterIdByName(unhealthyClusterScenario.clusterName),
+          []
+        );
+      });
+
+      it(`should have ${healthyClusterScenario.clusterName} displaying healthy state`, () => {
+        cy.get('td')
+          .contains(healthyClusterScenario.clusterName)
+          .parent()
+          .parent()
+          .prev()
+          .get('div > svg')
+          .should('have.class', 'fill-jungle-green-500');
+      });
+
+      it(`should have ${unhealthyClusterScenario.clusterName} displaying unhealthy state`, () => {
+        cy.get('td')
+          .contains(unhealthyClusterScenario.clusterName)
+          .parent()
+          .parent()
+          .prev()
+          .get('div > svg')
+          .should('have.class', 'fill-red-500');
+      });
+    });
+
+    describe('Clusters Tagging', () => {
+      before(() => {
+        cy.get('body').then(($body) => {
+          const deleteTag = 'span.ml-2.cursor-pointer';
+          if ($body.find(deleteTag).length > 0) {
+            cy.get(deleteTag).then(($deleteTag) =>
+              cy.wrap($deleteTag).click({ multiple: true })
+            );
+          }
+        });
+      });
+      const clustersByMatchingPattern = (pattern) => (clusterName) =>
+        clusterName.includes(pattern);
+      const taggingRules = [
+        ['hana_cluster_1', 'env1'],
+        ['hana_cluster_2', 'env2'],
+        ['hana_cluster_3', 'env3'],
+      ];
+
+      taggingRules.forEach(([pattern, tag]) => {
+        describe(`Add tag '${tag}' to all clusters with '${pattern}' in the cluster name`, () => {
+          availableClusters
+            .filter(clustersByMatchingPattern(pattern))
+            .forEach((clusterName) => {
+              it(`should tag cluster '${clusterName}'`, () => {
+                cy.get('td > span > a > span')
```

Comment:
The same here, we have a command for this:
https://github.com/trento-project/web/blob/main/test/e2e/cypress/support/commands.js#L92